kombu/utils/json.py

```diff
@@ -12,18 +12,18 @@ class DjangoPromise:
         """Dummy object."""
 
 try:
+    import json
+    _json_extra_kwargs = {}
+
+    class _DecodeError(Exception):
+        pass
+except ImportError:                 # pragma: no cover
```

Comment:
As far as I understand, ImportError will be raised only for python<2.6, which is not supported.

Maybe remove simplejson related code at all?